signing_clients/constants.py

```diff
@@ -0,0 +1,8 @@
+OU_PRODUCTION = 'Production'
+OU_PRIVILEGED_WEB_EXTENSION = 'Mozilla Extensions'
+OU_SYSTEM_ADDON = 'Mozilla Components'
+
+TYPE_HOTFIX = 'hotfix'
+TYPE_ADDON = 'add-on'
+TYPE_PRIVILEGED_WEB_EXTENSION = 'privileged web extension'
+TYPE_SYSTEM_ADDON = 'system add-on'
```

Comment:
I'm +1 for constants as a rule... but we're just transposing one constant for another.  If the constants are well named, do we really need to do: `if ou == constants.OU_SYSTEM_ADDON: constants.TYPE_SYSTEM_ADDON` ?